packages/react-compose/package.json

```diff
@@ -35,13 +35,21 @@
   },
   "dependencies": {
     "@types/classnames": "^2.2.9",
-    "@fluentui/set-version": "^8.0.0-beta.2",
-    "@fluentui/utilities": "^8.0.0-beta.15",
+    "@fluentui/set-version": "^8.0.0",
+    "@fluentui/utilities": "^8.0.0",
     "tslib": "^1.10.0",
     "classnames": "^2.2.6"
   },
   "peerDependencies": {
     "@types/react": ">=16.8.0 <18.0.0",
     "react": ">=16.8.0 <18.0.0"
+  },
+  "beachball": {
+    "tag": "beta",
+    "disallowedChangeTypes": [
+      "major",
+      "minor",
+      "patch"
+    ]
```

Comment:
This package will be removed soon:
- for converged components we will not use it (we have `react-utilities`)
- for v0 we will move `compose()` to `react-bindings` as v0 uses it multiple places

Reply:
This change means it won't be bumped. Otherwise it would go back to the default of bumping to non-beta versions and `latest` tag, which isn't what we want.

Reply:
Got it, resolved other comments related to `beachball` configs 👍